models/fixtures/attachment.yml

```diff
@@ -82,6 +82,7 @@
   id: 9
   uuid: a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a19
   repo_id: 1
+  issue_id: 2
```

Comment:
One attachment should not belong both an issue and a release

Reply:
Good catch, thanks!